ArduCopter/autoyaw.cpp

```diff
@@ -2,11 +2,19 @@
 
 Mode::AutoYaw Mode::auto_yaw;
 
+#define ROI_YAW_SCHEDULER_HZ 100 // ROI yaw should be calculated at 100Hz
```

Comment:
not blocking but I prefer static constexpr uint8_t here,

Reply:
done